File: backend/src/main/java/com/beenabler/backend/model/Entry.java
package com.beenabler.backend.model;

import lombok.With;

@With
public record Entry(
        String id,
        String beehiveId,
        String dateTime,
        String title,
        double weight,
        double feeding,
        double honeyHarvest,
        boolean varroaTreatment,
        boolean queen,
        boolean eggs,
        boolean brood,
        boolean queenCells
) {
}

File: frontend/src/types/Entry.ts
export type Entry = {
    id?: string,
    beehiveId?: string,
    dateTime?: string,
    title: string,
    weight: number,
    feeding: number,
    honeyHarvest: number,
    varroaTreatment: boolean,
    queen: boolean,
    eggs: boolean,
    brood: boolean,
    queenCells: boolean,
}